notebooks/BTA_array_producer.ipynb

```diff
@@ -0,0 +1,2322 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    # trk = events.PFCands[(events.PFCands.trkQuality != 0) &amp; (events.PFCands.pt &gt; 5.)]
</pre><p>if this code is really not needed anymore, i.e. if the JetPFCands collection is enough to build the TRK vars, please remove it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cms-btv-pog/BTVNanoCommissioning/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
not in the `BTA_producer.py`